control_plane/storcon_cli/src/main.rs

```diff
@@ -283,6 +291,24 @@ impl FromStr for PlacementPolicyArg {
     }
 }
 
+#[derive(Debug, Clone)]
+struct SkSchedulingPolicyArg(SkSchedulingPolicy);
+
+impl FromStr for SkSchedulingPolicyArg {
+    type Err = anyhow::Error;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        match s {
+            "active" => Ok(Self(SkSchedulingPolicy::Active)),
+            "disabled" => Ok(Self(SkSchedulingPolicy::Decomissioned)),
```

Comment:
Assuming that 'disabled' just means "don't do anything right now", let's call this `pause` for symmetry with tenant and node scheduling policies

Reply:
Yep, find to do this separately as a cleanup